src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const DAILY_RATE = 40;
+
+  const TOTAL_COST = DAILY_RATE * days;
```

Comment:
Consider renaming `TOTAL_COST` to `totalCost` as it is not a constant in the sense that its value is computed based on the input parameter `days`. By convention, uppercase names are used for constants whose value is static and known at compile-time, while camelCase is used for variables whose values are computed during execution.